lib/docs/filters/core/home_url.rb

```diff
@@ -0,0 +1,16 @@
+module Docs
+  class HomeUrlFilter < Filter
+    def call
+      html.prepend(home_url_html) if home_url
```

Comment:
> I'm not sure what you wanted changed in `doc.rb`. Is that `lib/docs/core/doc.rb` that you meant?

Yes, I meant here:
https://github.com/Thibaut/devdocs/blob/master/lib/docs/core/doc.rb#L7
This is an attribute that all docs will have, scraper or not.